db/schema.rb

```diff
@@ -1,36 +1,23 @@
-# This file is auto-generated from the current state of the database. Instead
-# of editing this file, please use the migrations feature of Active Record to
-# incrementally modify your database, and then regenerate this schema definition.
-#
-# This file is the source Rails uses to define your schema when running `bin/rails
-# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
-# be faster and is potentially less error prone than running all of your
-# migrations from scratch. Old migrations may fail to apply correctly if those
-# migrations use external dependencies or application code.
-#
-# It's strongly recommended that you check this file into your version control system.
-
-ActiveRecord::Schema[7.0].define(version: 2022_03_25_203647) do
-  # These are extensions that must be enabled in order to support this database
+ActiveRecord::Schema[7.0].define(version: 2022_03_27_232532) do
   enable_extension "plpgsql"
 
   create_table "comments", force: :cascade do |t|
     t.text "text"
-    t.bigint "author_id"
-    t.bigint "post_id"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.index ["author_id"], name: "index_comments_on_author_id"
+    t.bigint "user_id", null: false
+    t.bigint "post_id", null: false
     t.index ["post_id"], name: "index_comments_on_post_id"
+    t.index ["user_id"], name: "index_comments_on_user_id"
   end
 
   create_table "likes", force: :cascade do |t|
-    t.bigint "author_id"
-    t.bigint "post_id"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.index ["author_id"], name: "index_likes_on_author_id"
+    t.bigint "user_id", null: false
```

Comment:
- [ ] I suggest we go back to `author_id` because that is how the ERD shows, see the image below. We can also change the model associations to match the design. For example, `belongs_to :author, class_name: 'User'`

![authorid](https://user-images.githubusercontent.com/53124548/160345607-0748b101-4e2d-40ff-a52b-240c2fefff10.png)